doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -366,8 +366,154 @@ For example, you can use the following command when building on the command line
 
 To build for the nRF7002 EK and the nRF7001 or nRF7000 ICs, you can use the corresponding shield name in the above command.
 
+nRF7002 EB
+==========
+
+The nRF7002 :term:`Expansion Board (EB)` can be used to provide Wi-Fi connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
+
+The nRF7002 EB has a :term:`Printed Circuit Board (PCB)` edge connector that can be used with a compatible development board such as the Nordic Thingy:53, an IoT prototyping platform from Nordic Semiconductor.
+There are also castellated holes on the side of the board that allow the EB to be used as a breakout board that can be soldered to other PCB assemblies.
+
+Overview
+--------
+
+The nRF7002 EB (PCA63561) features a PCB edge connector and castellated holes to provide Wi-Fi connectivity through the nRF7002 companion IC.
+
+The PCB edge connector is used to provide Wi-Fi connectivity to a development board with a compatible connector such as the Nordic Thingy:53, where it connects nRF7002 to nRF5340, which acts as a host.
+The EB can also be used to provide Wi-Fi capabilities to develop Wi-Fi applications with another System on Chip (SoC), MPU, or MCU host by using the castellated mounting holes on the sides of the board.
+
+.. figure:: images/nRF7002eb.png
+   :alt: nRF7002 EB
+
+   nRF7002 EB
+
+Pin assignment
+--------------
+
```

Comment:
I believe there should be a sentence here, even if brief as we tend to avoid subheadings following immediately after a main heading.

I see however that that was done previously on line 288.